S21/FaizanSyed95/Language_test.py

```diff
@@ -0,0 +1,60 @@
+import csv 
+
+Vowels=['A','E','I','O','U','Á','É','Í','Ó','Ú']
+Greek=['Π','Ό','Λ','Η','Σ']
+Greek_again=['π','ό','λ','η','ς']
+
+
+def Word_check(lang , word):
+    LG=list(lang)
+    
+    if LG[0]=='z' and LG[1]=='h' or LG[0]=='J' and LG[1]=='a' or LG[0]=='t' and LG[1]=='h':
+        return word
+    
+            
+    elif LG[0]  == "e" and LG[1] == "n" :
+            return word.lower()
+        
+    elif  LG[0]=='t' and LG[1]=='r' or LG[0]=='a' and LG[1]=='z'  :
+            wordArray= list(word)     
+            for i in range(len(wordArray)):   
+                if wordArray[i] == "I":
+                     wordArray[i] = "ı"
+                     
+            return ''.join(wordArray).lower()
+        
+    elif  LG[0]=='g' and LG[1]=='a' :
+        wordArray = list(word)
+        for j in range(len(Vowels)):
+            if wordArray[0] == "t" or wordArray[0] == "n":
+                if Vowels[j] in wordArray[1]:
+                    wordArray.insert(1,'-')     
+        
+        return ''.join(wordArray).lower()
+   
+    elif LG[0]=='e' and LG[1]=='l':
+        wordArray=list(word)        
+        for i in range(len(wordArray)):
+            for j in range(len(Greek)):
+                if wordArray[i] == Greek[j]:
+                    wordArray[i] = Greek_again[j]
+                    
+        return ''.join(wordArray)
+        
+
+
+
```

Comment:
Instead of using the tests.tsv file directly, it may be better to use the input function for word and the language. Your function is nicely written, it would be able to decode the individual characters of the input.

Reply:
The reason for using the test file is, if we have many inputs for our test, it would be easier to write them all in one file, just like the professor gave us, and run them and check if these outputs match or not. My code checks all the list of inputs of word and languages and compare it with the output of the test file and gives the result.